networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/discovery/ConnectionManagerTest.java

```diff
@@ -80,23 +83,25 @@ public void setUp() {
   }
 
   @Test
-  public void shouldConnectToStaticPeersOnStart() {
+  public void shouldConnectToStaticPeersOnStart()
+      throws InterruptedException, ExecutionException, TimeoutException {
     final ConnectionManager manager = createManager(PEER1, PEER2);
     when(network.connect(any(PeerAddress.class))).thenReturn(SafeFuture.completedFuture(null));
-    manager.start().join();
+    Waiter.waitFor(manager.start());
```

Comment:
If we expect the future to already be completed it can be clearer to use
```
    assertThat(manager.start()).isCompleted();
```

Both work though and are both better than .join.